src/js/views/forms/form/form.scss

```diff
@@ -121,13 +121,14 @@
 .form__controls {
   display: flex;
   justify-content: flex-end;
-  margin-top: 12px;
+  line-height: 1.2;
+  padding: 16px 0 16px 16px;
 }
 
 .form__actions {
-  border-right: 1px solid $black-80;
   display: flex;
-  margin-right: 20px;
+
+  /* margin-right: 20px; */
```

Comment:
doesn't seem like this class is needed.  just swap it out with the flex utility class

Reply:
Good call, missed that one. Updated in last push.